google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadStoreSalesTransactions.java

```diff
@@ -257,7 +257,10 @@ private String createOfflineUserDataJob(
             // Continuing the example above for loyalty fraction, a value of 1.0 here indicates that
             // you are uploading all 70 of the transactions that can be identified by an email
             // address or phone number.
-            .setTransactionUploadFraction(DoubleValue.of(1.0));
+            .setTransactionUploadFraction(DoubleValue.of(1.0))
```

Comment:
Instead of this approach, please add command line support for the key via the following steps:

1. Add a new annotated, optional field on `UploadStoreSalesTransactionsParams`. This will require a new constant in `ArgumentNames.java` as well.
1. Set the field in the `if (!params.parseArguments(args))` block above.

This is the standard approach we use in all our examples, and it makes it easier for users.

* Those who want to pass everything on the command line without editing can do so.
* Those who want to edit the example only have to edit one section.

Thanks

Reply:
Thanks jradcliff. I made the changes.